src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/Cas1AutoScript.kt

```diff
@@ -56,6 +67,8 @@ class Cas1AutoScript(
         val roles = user.roles.map { it.role }.joinToString(", ")
         seedLogger.info("  -> User '${user.name}' (${user.deliusUsername}) seeded with roles $roles")
```

Comment:
I was thinking it might be helpful if we also logged `SeedUser.documentation` here to give a message like:

> -> User Fred Bloggs (Fred.Bloggs) seeded with roles UserRole.CAS1_MANAGER - For local use in development and testing